ansible/vars/topo_t0-56-o8v48.yml

```diff
@@ -106,6 +106,8 @@ topology:
         - 31
       vm_offset: 7
   DUT:
+    autoneg_interfaces:
+      intfs: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
```

Comment:
This change hardcodes interfaces and remove ability set interface 1,5,7 with autoneg enabled on 1 setup with t0-56 and 2,5,7,8 etc on other setup

Reply:
fixed, we dont need to add interfaces in this file anymore